static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -198,8 +195,40 @@ function OnboardingContent({currentProject}: {currentProject: Project}) {
   );
 }
 
+// TODO: we'll have to move this into a folder for common consumption w/ Profiling, Performance etc.
+interface OnboardingStepV2Props {
+  content: string;
+  step: number;
+}
+
+function OnboardingStepV2({step, content}: OnboardingStepV2Props) {
+  return (
+    <Flex>
+      <div>
+        <TaskStepNumber>{step}</TaskStepNumber>
+      </div>
+      <Flex.Item>
+        <DocumentationWrapper dangerouslySetInnerHTML={{__html: content}} />
+      </Flex.Item>
+    </Flex>
+  );
+}
+
+const TaskStepNumber = styled('div')`
+  display: flex;
+  margin-right: ${space(1.5)};
+  background-color: ${p => p.theme.yellow300};
+  border-radius: 50%;
+  font-weight: bold;
+  height: ${space(4)};
+  width: ${space(4)};
+  justify-content: center;
+  align-items: center;
+`;
```

Comment:
I've seen this component used in other places in Sentry, is there no existing component?

Reply:
its similar to our avatars, can't seem to find anything else